src/tests/PersonalDetails.test.js

```diff
@@ -95,3 +96,60 @@ it('should send modified fields in payload to update', async () => {
     fireEvent.click(screen.getByRole("button", {name: "Save"}), {target: {value: true}})
   });
 });
+
+it('checks empty field warning', async () => {
+
+  render(<PersonalDetails />)
+
```

Comment:
Please remove the extra spaces and just keep single line space where-ever required.